### PR TITLE
Changes for issue #773

### PR DESCRIPTION
**Pull request by joergboe**

### CODE REVIEW

**Comment by ddebrunner** [comment]
Can one of the admins verify this patch?

File: test/python/build.xml [hunk 1]
```diff
@@ -92,4 +92,9 @@
          </ant>
      </target>
 
+    <target name="test.translation" description="Run the translation tests">
+        <exec executable="/bin/bash" dir="spl/translation">
```
Comment (ddebrunner) [inline test/python/build.xml]: Should this have the `failonerror` property set to `true`?

**Comment by joergboe** [comment]
I changed the code accordingliy

**Comment by ddebrunner** [comment]
test now

**Pull request merged**